### PR TITLE
Change ansible-status to use boolean values.

### PR DESCRIPTION
**Pull request by marshall-mcmullen**
Change new `ansible_status` to use boolean option flags instead of strings.

### CODE REVIEW

File: share/ansible.sh [hunk 1]
```diff
@@ -14,9 +14,14 @@ END
 ansible_status()
 {
     $(opt_parse \
-        "=failed    f   | Did this module fail - true | false."             \
-        "=changed   c   | Did this module change anything - true | false."  \
-        "=msg       m   | Message associated with this status.")
+        "+failed      f | Did this module fail."                \
```
Comment (thecodenomad) [inline share/ansible.sh:17]: Nice ;)

**Review by thecodenomad — APPROVED** [review]
I dig it!

**Review by bowanggu — APPROVED** [review]

File: share/ansible.sh (hunk 1 above)
Comment (kirby-michael) [reply share/ansible.sh:17]: Thank you!

File: share/ansible.sh [hunk 2]
```diff
@@ -14,9 +14,14 @@ END
...
+        "+changed     c | Did this module change anything."     \
+        "=message msg m | Message associated with this status." \
+    )
+
+    # Convert boolean values in failed and changed to string values
+    failed=$(bool_to_string "${failed}")
+    changed=$(bool_to_string "${changed}")
```
Comment (bowanggu) [inline share/ansible.sh:24]: @marshall-mcmullen on line 28, needs to change to:  msg=$(json_escape "${message}")

**Pull request merged**